lemmatize_text_data_WIP.py

```diff
@@ -0,0 +1,43 @@
+from settings import DATA_DIR, MODEL_DIR
```

Comment:
We usually do not put WIP into file name.
If we want to suggest WIP, we put to the function in progress:
 - raise NotImplementedError
 -  or # FIXME comment